common/fuse.h

```diff
@@ -296,6 +296,11 @@ class Fusor : public Looper {
     return sqrtf(gyro_slope_.z * gyro_slope_.z + gyro_slope_.y * gyro_slope_.y) * (M_PI / 180);
   }
 
+  // Acceleration into twist (one direction) in radians per second per second
+  float twist_accel() {
+    return sqrtf(gyro_slope().x * gyro_slope().x);
```

Comment:
Slope is naturally a function that can be both positive and negative.
Using fabs() isn't a solution, it just confuses things.
Even if we do need fabs() in the style, it should go there, not here.

Reply:
ok, moved fabs() into the function.

Reply:
I'm still not convinced that we need/want a fabs()
Do you not care if the action is speeding up or slowing down?

Reply:
Since all other functions, SwingSpeed, BladeAngle, SwingAcceleration, etc return 0 ~ 32768 I kept this in the same vein.  In testing, particularly if used in Shapes, negative numbers created very odd movement or results.
Nearly all of my intended uses are looking for the acceleration of the twist action or the acceleration up to it, not really slowing down.  
I suppose we can remove the fabs() but we may want to add an Absolute<> function separately so it can be used in Shapes or other instances where a negative number causes issues or where the slowing down is unwanted.